packages/harpparts/src/types/types.ts

```diff
@@ -12,3 +12,13 @@ export type HalfstepIndex = number
 
 export type HarpFaceRow<T> = ReadonlyArray<T | undefined>
 export type HarpFaceMatrix<T> = ReadonlyArray<HarpFaceRow<T>>
+type HarpFaceMatricesDiatonic<T> = {
```

Comment:
This is the file where we perhaps need to consider the creation of a `PossiblePair` type, which can either be of HarpFaceMatrix, or of ReedArray or other types..